core/templates/pages/exploration-editor-page/settings-tab/settings-tab.component.html

```diff
@@ -184,7 +184,7 @@ <h3>Advanced Features</h3>
     </div>
 
     <div ng-if="$ctrl.advancedFeaturesIsShown" class="oppia-mobile-collapsible-card-content">
-      <div role="form" class="form-horizontal">
+      <div role="form" class="form-horizontal" ng-if="$ctrl.areParametersEnabled()">
```

Comment:
Hi @aiswary023, did you test if it is possible to turn params off?

I was not able to do so. PTAL
![notclickable](https://user-images.githubusercontent.com/11008603/103452304-e6549980-4cf3-11eb-88ce-9c86dceee914.gif)

Reply:
Hi @kevintab95 , I took a look and found that there is no function to disable parameters. Currently, parameters can only be enabled. Is this supposed to be this way  or should I create a function to disable parameters?

Reply:
Hm, interesting, can you try tracing back to when the feature was introduced to see if this was intentional (and if there are any caveats in disabling it)?

Reply:
Hi Kevin, sorry for the late response. According to [this comment](https://github.com/oppia/oppia/pull/4811#issuecomment-374690018) parameters switch is supposed to be one-way.

Reply:
Thanks @aishwary023! Great job btw for finding out the reasoning for the behaviour! :)